Vostok.Metrics.System/Host/HostMetrics.cs

```diff
@@ -83,27 +84,37 @@ public class HostMetrics
         /// </summary>
         public Dictionary<TcpState, int> TcpStates { get; set; }
 
+        /// <summary>
+        /// Network usage info per interface.
+        /// </summary>
+        public Dictionary<string, NetworkInterfaceUsageInfo> NetworkInterfacesUsageInfo { get; set; }
+
         /// <summary>
         /// Amount of network sent bytes per second (across all interfaces).
         /// </summary>
-        public long NetworkSentBytesPerSecond { get; set; }
+        public long NetworkSentBytesPerSecond => NetworkInterfacesUsageInfo.Sum(x => x.Value.SentBytesPerSecond);
 
         /// <summary>
         /// Amount of network received bytes per second (across all interfaces).
         /// </summary>
-        public long NetworkReceivedBytesPerSecond { get; set; }
+        public long NetworkReceivedBytesPerSecond => NetworkInterfacesUsageInfo.Sum(x => x.Value.ReceivedBytesPerSecond);
+
+        /// <summary>
+        /// Maximum host network speed (across all interfaces).
+        /// </summary>
+        public long NetworkBandwidthBytesPerSecond => NetworkInterfacesUsageInfo.Sum(x => x.Value.BandwidthBytesPerSecond);
 
         /// <summary>
         /// <para>Utilized percent of the output network bandwidth (relative to all interfaces).</para>
         /// <para>This metric is an average value between two observation moments (current and previous).</para>
         /// </summary>
-        public double NetworkOutUtilizedPercent { get; set; }
+        public double NetworkOutUtilizedPercent => (NetworkSentBytesPerSecond * 100d / NetworkBandwidthBytesPerSecond).Clamp(0, 100);
```

Comment:
Avoid division by zero when no network interfaces are detected?

Reply:
We have double division there. `Math.Min(0, double.NaN)` equals 0, so if there are no network interfaces we will have correct value.